jaegertracing/jaegertracing.go

```diff
@@ -62,6 +62,9 @@ type (
 		// http body limit size (in bytes)
 		// NOTE: don't specify values larger than 60000 as jaeger can't handle values in span.LogKV larger than 60000 bytes
 		LimitSize int
+
+		// OperationNameFunc composes operation name based on context. Can be used to override default naming
+		OperationNameFunc func(ctx echo.Context) string
```

Comment:
Please rename that parameter from `ctx` to `c`. Named parameters are handy when your IDE (for example GoLand by Jetbrains) does autocompletion and helps to autofill names. By convention `echo.Context` is usually named as `c`.

Reply:
Done